src/common/safeIpc.ts

```diff
@@ -1,13 +1,4 @@
-import {
-    BrowserWindow,
-    IpcMainEvent,
-    IpcMainInvokeEvent,
-    IpcRendererEvent,
-    MessagePortMain,
-    WebContents,
-    ipcMain as unsafeIpcMain,
-    ipcRenderer as unsafeIpcRenderer,
-} from 'electron';
+import { type FileFilter, type OpenDialogReturnValue } from 'electron'; // TODO: replace with electron/common
```

Comment:
TODO

Reply:
That's not a now todo, that's a "we can't do that til we do more stuff in the future" todo. As in, if I were to do that now this wouldn't compile.

Reply:
I also explained this in point 2 of my comment. For reference:

> This means that i had to leave a few as regular electron imports in the meantime, until we swap over to exposing IPC via the preload script